docs/src/markdown-pages/guides/getting-started.md

```diff
@@ -7,23 +7,23 @@ description: 'How to get moving running and developing your first loop.'
 # Getting Started
 
 1. Download Olive Helps from the root page of [the Developer Hub](https://open-olive.github.io/olive-helps/).
-2. Install and boot up Olive Helps.
-3. Create an Olive account, and log in.
-4. Open the Olive Helps drawer and open the App Menu in the top right.
-5. Go to the Loop Library.
-6. Click the link to "Become a Loop Author".
-7. Click the button to "Make me a Loop Author!".
-8. Agree to the Terms and Conditions.
+1. Install and boot up Olive Helps.
+1. Create an Olive account, and log in.
+1. Open the Olive Helps drawer and open the App Menu in the top right.
+1. Go to the Loop Library.
+1. Click the link to "Become a Loop Author".
+1. Click the button to "Make me a Loop Author!".
+1. Agree to the Terms and Conditions.
 
 And congrats! You've officially signed up to become a Loop author. The next thing you'll want to do is to install a Loop locally. Some examples have been included in the LDK for you to use.
 
 1. Download the LDK from [our Github](https://github.com/open-olive/loop-development-kit). Alternatively, you could copy one of the example Loop directories
-2. Run Olive Helps, and then go to the Loop Libraray
-3. Click on "Local Loops"
-4. Click "Install Local Loop"
-5. Choose the directory of the example you wanted to install in the picker.
-6. Configure the information about your local Loop, and click Install. If everything goes well, you will get a notoification that it succeeded.
-
-NOTE: For the "Command" value, you will need to put in whatever the terminal command is to run your main function. For example, for the Node examples in the LDK, the command is "./node_modules/.bin/ts-node src/index.ts"
-
-7. Test out the Loop, and then make some changes. You can then press "Restart" in the Loop Libraray to pull in these changes.
+    - These are located within the LDK at `ldk/javascript/examples`
+1. Build your local Loop. See [these instructions](https://github.com/open-olive/loop-development-kit/tree/main/ldk/javascript#producing-loop-compilations) for more information.
+1. Run Olive Helps, and then go to the Loop Library
+1. Click on "Local Loops"
```

Comment:
We should establish a style guide that outlines how to highlight user elements. I've personally preferred — and just learned now that [Google does the same thing](https://developers.google.com/style/ui-elements) — to bold element labels instead of quote them.